src/D2L.Security.OAuth2/Keys/Default/Data/JwksProvider.cs

```diff
@@ -70,7 +75,7 @@ Uri authEndpoint
 			}
 		}
 
-		string IJwksProvider.Namespace => m_authEndpoint.AbsoluteUri;
+		string IJwksProvider.Namespace => m_jwksEndpoint.AbsoluteUri;

```

Comment:
@omsmith FYI. @khorwood and I discussed this in person. There are some implicit assumptions here but seems reasonable?